File: Prueba y error/ejercicios de semana santa php/Four php/Library/LicenciaA.php
<?php
require_once 'Licencia.php';

class LicenciaA extends Licencia {
    public function __construct() {
        $this->tipo = "Licencia A (Motocicletas)";
        $this->duracion = 12;
        $this->costo = 150000;
    }
}

File: Prueba y error/ejercicios de semana santa java/four java/Library/LicenciaA.java
package Library;

import java.text.DecimalFormat;

public class LicenciaA extends Licencia {
    public LicenciaA() {
        this.tipo = "LicenciaA (Motocicletas)";
        this.duracion = 12;  // 
        this.costo = 250000;  // (COP)
    }

    @Override
    public String detallesLicencia() {
         // decimales . y , al valor de la conosola
        DecimalFormat formato = new DecimalFormat("#,###.00");
        String costoFormateado = formato.format(costo);
        return "Tipo de Licencia: " + tipo + "\nDuración: " + duracion + " meses\nCosto: $" + costoFormateado + " COP";
    }
}